timescaledb/tutorials/grafana/visualizations/bar-chart.md

```diff
@@ -0,0 +1,201 @@
+import GrafanaVizPrereqs from 'versionContent/_partials/_grafana-viz-prereqs.mdx';
+
+# Build a bar chart in Grafana
+
+A bar chart displays data using bars, each representing a specific category. A
+bar chart is a great comparison tool for data visualization, as it's easy to see
+which item's column or bar is longer or shorter. Bar charts use two axes:
+vertical and horizontal. The longer a bar is, the greater its value. A bar chart
+is a good way to compare items between different groups.
+
+Bar charts can answer questions like:
+
+* Which stock has the highest traded volume today?
+* What was the transaction volume distribution of stocks last week?
+* How many students in grade A are above a certain age range?
+
+<!--- Do you have a diagram to use here? --LKB 2022-07-01-->
+
+To plot a bar chart, Grafana requires only one data frame. You need to have at
+least one string field, which is used as the category for an X or Y axis, and
+one or more numerical fields. You can also provide multiple value columns, if
+you want to plot multiple bar charts in a single panel.
+
+This tutorial shows you how to:
+
+* [Create a bar chart](https://docs.timescale.com/timescaledb/latest/tutorials/grafana/visualizations/bar-chart/#create-a-bar-chart-with-preaggregated-data)
+  with pre-aggregated data using `time_bucket()`.
+* [Create multiple bar charts](https://docs.timescale.com/timescaledb/latest/tutorials/grafana/visualizations/bar-chart/#create-multiple-bar-charts)
+  in a single panel.
+* [Create a stacked bar chart](https://docs.timescale.com/timescaledb/latest/tutorials/grafana/visualizations/bar-chart/#create-stacked-bar-chart)
+  with pre-aggregated data.
+
+There are a few different kinds of bar charts to choose from, including
+vertical, horizontal, and stacked bar charts. All of these types are covered in
+this tutorial.
+
+## Prerequisites
+
+<GrafanaVizPrereqs />
+
+## Create a bar chart with preaggregated data
+
+Create a bar chart visualization using the data in the table `stocks_real_time`.
+
+<procedure>
+
+### Creating a bar chat with preaggregated data
+
+1. In the query editor, use this SQL to query a bar chart dataset. Use the
+    variable `$bucket interval` for the time period covered by the bar chart:
+
+    ```sql
+    SELECT
+        time_bucket('$bucket_interval', time) AS time,
+        symbol,
+        AVG(price) as price
+    FROM stocks_real_time srt
+    WHERE symbol = $symbol
+        AND time >= $__timeFrom()::timestamptz AND time < $__timeTo()::timestamptz
+    GROUP BY time_bucket('$bucket_interval', time), symbol;
+    ```
+
+1. In the Grafana dashboard, in the `Dashboard variable` field, select a stock
+    to graph. Adjust the time range of the dashboard if needed. Make sure the
+    returned data has column named `time` that contains timestamps. The
+    timestamps should be in ascending order, you might get errors otherwise. The
```

Comment:
```suggestion
    returned data has a column named `time` that contains timestamps. The
    timestamps should be in ascending order. Otherwise, you get an error. The
```